src/test/java/org/apache/sysml/test/integration/AutomatedTestBase.java

```diff
@@ -95,6 +95,7 @@ public String lowerCase() {
 	
 	// By default: TEST_GPU is set to false to allow developers without Nvidia GPU to run integration test suite 
 	public static final boolean TEST_GPU = false;
+	public static final double GPU_TOLERANCE = 1e-6;
```

Comment:
@niketanpansare - this is the threshold that i used for my tests. But I'd recommend using as **small** a threshold as possible (maybe start with 10e-10?)